vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -58,7 +58,9 @@ function Get-ContainerImages {
                 "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v2.0.1",
                 "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v0.5.1",
                 "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v0.6.0",
-                "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v0.7.0")
+                "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v0.7.0",
+                "mcr.microsoft.com/oss/kubernetes-csi/secrets-store/driver:v0.0.19",
+                "mcr.microsoft.com/oss/azure/secrets-store/provider-azure:0.0.12")
```

Comment:
@ZeroMagic , you need also to update the test https://github.com/Azure/AgentBaker/blob/master/vhdbuilder/packer/test/vhd-content-test.ps1#L175 accordingly, Thanks.

Reply:
done